src/main.cpp

```diff
@@ -136,7 +119,24 @@ static void get_options(int argc, char **argv, po::variables_map &options) {
   desc.add(expert);
 
   po::store(po::parse_command_line(argc, argv, desc), options);
-  po::store(po::parse_environment(desc, environment_option_name), options);
+  po::store(po::parse_environment(desc,
+    [desc](const std::string &name) {
```

Comment:
Maybe capture desc by reference instead?
```
  po::store(po::parse_environment(desc,
    [&desc](const std::string &name) {
```

Reply:
Good point, no need to pass by value there.